test_test.go

```diff
@@ -17,7 +17,7 @@ func TestTest(t *testing.T) {
 	config.Complexity = 0
 	config.Repeat = 1
 
-	test, err := NewTest(config)
+	test, err := NewTest(config, PromConfig{}, CIConfig{})
```

Comment:
this is making tests fail, constructor no longer takes prometheus and ci config

Reply:
ouch, typo, wait a sec

Reply:
Done